package.json

```diff
@@ -15,9 +15,14 @@
     "babel-preset-env": "^1.6.1",
     "babel-preset-stage-2": "^6.24.1",
     "babel-register": "^6.26.0",
+    "eventemitter2": "^5.0.1",
```

Comment:
Please, use built-in `events` NodeJS package for EventEmitter

Reply:
Done